news/_posts/2020-09-10-bug-bounty-retrospective.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: post
+title:  "Bug Bounty Program, the retrospective"
+subtitle: "When hunters make your business safer!"
+date:   2020-09-10 09:00:00
+authors: [ PierreRambaud, PrestaShop ]
+icon: icon-chat
+image: /assets/images/2020/09/ywhbanner.jpg
+tags: [bug, bounty, program, maintenance]
+---
+
+During months, we experimented a Bug Bounty Program with the [YesWeHack](https://yeswehack.com) interface. Here's the retrospective about why we did this, and what happened during this period.
+
+
+## Private program
+
+We never really know the security of our software before it has been tested by a lot of hunters.
+
+PrestaShop is full of dependencies, with hundred of thousands lines of code, and for most of them exist since the first version. 
+
+Using a public program directly would have been a too big risk for us, causing a huge charge of duplicated reports and creating frustration among hunters.
+
+The benefit of the private program is to limit the number of hunters and manage the reporting flow. Because on each security issue, you need to take in consideration the time to understand, reproduce, fix, test by the QA Team, etc. 
```

Comment:
```suggestion
The benefit of the private program is to limit the number of hunters and manage the reporting flow. For each reported security issue, resources must be dedicated to understand it, reproduce it, build a patch, test the patch by the QA Team, etc. This can be quite expensive, so it is important to be able to adjust how many issues are reported, thanks to the private program settings.
```